components/datetime/src/datetime.rs

```diff
@@ -126,9 +126,9 @@ impl DateTimeFormat {
                 data_provider
                     .load_payload(&DataRequest {
                         resource_path: ResourcePath {
-                            key: provider::key::GREGORY_DATE_SYMBOLS_V1,
+                            key: provider::key::DATE_SYMBOLS_V1,
                             options: ResourceOptions {
-                                variant: None,
+                                variant: Some("gregorian".into()),
```

Comment:
Nit: We should stick with the BCP-47 names (i.e. "gregory") so that we have one set of names.

https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-bcp47/bcp47/calendar.json